kiwi/bootloader/config/base.py

```diff
@@ -298,12 +299,17 @@ def get_boot_path(self, target='disk'):
                 filesystem = self.xml_state.build_type.get_filesystem()
                 volumes = self.xml_state.get_volumes()
                 if filesystem == 'btrfs' and volumes:
-                    # boot or boot/grub2 on a subvolume prevents grub from
-                    # finding its config file
+                    # grub boot data paths must not be in a subvolume
+                    # otherwise grub won't be able to find its config file
+                    grub_boot_data_paths = [
+                        'boot', os.sep.join(['boot', 'boot/grub', 'boot/grub2'])
```

Comment:
right, that happens if two thoughts come together where only one was correct :) Thanks for finding this